.circleci/config.yml

```diff
@@ -6,8 +6,10 @@ jobs:
     environment:
       CIRCLE_ARTIFACTS: ~/projectcalico/calicoctl/bin
     steps:
+    - run: |
+          mkdir -p /home/circleci/.go_workspace/pkg/mod
```

Comment:
```suggestion
          # pre-create the go mod cache directory. otherwise, docker will create it with root ownership
          mkdir -p /home/circleci/.go_workspace/pkg/mod
```